plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1013,6 +1027,7 @@ void producer_plugin::plugin_startup()
    my->_accepted_block_connection.emplace(chain.accepted_block.connect( [this]( const auto& bsp ){ my->on_block( bsp ); } ));
    my->_accepted_block_header_connection.emplace(chain.accepted_block_header.connect( [this]( const auto& bsp ){ my->on_block_header( bsp ); } ));
    my->_irreversible_block_connection.emplace(chain.irreversible_block.connect( [this]( const auto& bsp ){ my->on_irreversible_block( bsp->block ); } ));
+   my->_block_start_connection.emplace(chain.block_start.connect( [this]( uint32_t bs ){ my->_snapshot_scheduler.on_block(bs); } ));
```

Comment:
Please use `on_start_block` for `block_start` signal to avoid confusion. We use `on_block` for `accepted_block` signal. Although, I think you want to attach to `accepted_block` not `block_start`.

Reply:
done